frontend/packages/console-shared/src/components/badges/Badge.scss

```diff
@@ -0,0 +1,6 @@
+.ocs-badge {
+  &__inline {
+    background-color: var(--pf-global--BackgroundColor--300);
+    border: 1px solid var(--pf-global--BorderColor--light-100);
```

Comment:
Interesting... was there an issue with the pf colour variables provided by UX?

> The badge in the proposed design has background colour "pf-colour-black-200 (#EDEDED)" and border colour "pf-colour-black-400(#B8BBBE)"

Reply:
I didn't find any variable with those names https://patternfly-react.surge.sh/documentation/overview/global-css-variables . but the variable I uses has the same values provided by UX, also seemed like a good choice to use the global variable, and these are specifically for border and background.

Reply:
Interesting. We should get UX to align with these variable availability otherwise them providing a pf variable name isn't going to be all that helpful.

Thoughts @christianvogt?

cc @parvathyvr

Reply:
I think PF would suggest using the global variables.
I believe to get access to the variables UX specified would require using the SCSS vars.

Reply:
Interesting... 🤔 We may need to get all aligned here otherwise we're going to run into situations were we have to "go find" a different variable or move away from the CSS variables to the SCSS variables.

Reply:
@andrewballantyne @sahil143 When i took the color name i referenced it from the sketch library and it just mentions 'pf-colour-balck-200'. I discussed this confusion regarding CSS variables with the team and found out that this is the correct link to refer the Global CSS variables https://www.patternfly.org/v4/documentation/overview/global-css-variables#component-title

Reply:
Oooh, most interesting. Thanks for that share @parvathyvr.